config/webhook/manifests.yaml

```diff
@@ -26,7 +25,6 @@ webhooks:
         resources:
           - inferenceservices
   - clientConfig:
-      caBundle: Cg==
```

Comment:
Is the placeholder not needed anymore ?

Reply:
Yes. It is not used anywhere. The caBundle is automatically injected by the cert-manager. Having this placeholder produces conflict when we try to run the `make deploy-dev` second time as it is trying to replace the `ca bundle` with this dummy value.